src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -25,6 +27,16 @@ export class ArtworkSidebarBidAction extends React.Component<
     this.setState({ nextMaxBidCents: newVal })
   }
 
+  redirectToLiveBidding = (user: User) => {
+    const { id } = this.props.artwork.sale
+    const liveUrl = `${sd.PREDICTION_URL}/${id}`
+    if (user) {
```

Comment:
I took this logic from https://github.com/artsy/force/blob/0a4ba5c3327c0397a0b63f7a64dfd0bb5617bf06/src/utils/domain/auctions/urls.js#L14-L15 , but it reads strangely to me.

If the user is logged in (to Force), then bring them to `/login` in Prediction? Does that do something behind the scenes to seamlessly log them in, vs without that it's something else?

Reply:
Yup, it will check for an access token and then do a redirect, leading them to a biddable live auction.